packages/webdriverio/src/index.ts

```diff
@@ -0,0 +1,230 @@
+import type { RunOptions, AxeResults } from 'axe-core';
+import { source } from 'axe-core';
+import * as assert from 'assert';
+import * as cssesc from 'cssesc';
+import type {
+  Options,
+  CallbackFunction,
+  BrowserObject,
+  Element
+} from './types';
+import {
+  analyzePage,
+  isWebdriverClient,
+  normalizeContext,
+  logOrRethrowError
+} from './utils';
+
+export default class AxeBuilder {
+  private client: BrowserObject;
+  private axeSource: string;
+  private includes: string[];
+  private excludes: string[];
+  private option: RunOptions;
+  private disableFrameSelectors: string[];
+  constructor({ client }: Options) {
+    assert(
+      isWebdriverClient(client),
+      'An instantiated webdriverio client of v5 or v6 is required.'
```

Comment:
can we say this - https://github.com/dequelabs/axe-devtools-npm/blob/develop/packages/webdriverio/src/index.ts#L107

Given newer version 7, is coming out soon?